templates/vue/src/components/tyde/TydeCockpit.vue

```diff
@@ -126,4 +147,5 @@ export default {
     overflow-y: scroll;
   }
 }
+
 </style>
```

Comment:
Please keep the original TydeMenuHome component. We may want to use it still in the future.